data/antivirus.yaml

```diff
@@ -105,18 +115,23 @@ sources:
     paths:
     - '%%environ_allusersappdata%%\Symantec\Symantec Endpoint Protection\*\Data\Logs\*.log'
     - '%%environ_allusersappdata%%\Symantec\Symantec Endpoint Protection\*\Data\Logs\AV\*.log'
+    - '%%environ_allusersappdata%%\Symantec\Symantec Endpoint Protection\Logs\AV\*.log'
     - '%%users.localappdata%%\Symantec\Symantec Endpoint Protection\Logs\*.log'
     separator: '\'
   supported_os: [Windows]
 supported_os: [Windows]
 labels: [Antivirus, Logs]
 ---
 name: SymantecAVQuarantine
-doc: Symantec Anti-Virus Quarantine (Infected) files.
+doc: Symantec Anti-Virus quarantine (infected) and cloud submission files.
 sources:
 - type: FILE
   attributes:
-    paths: ['%%environ_allusersappdata%%\Symantec\Symantec Endpoint Protection\**5\*.vbn']
+    paths:
+    - '%%environ_allusersappdata%%\Symantec\Symantec Endpoint Protection\**5\*.vbn'
+    - '%%environ_allusersappdata%%\Symantec\Symantec Endpoint Protection\Quarantine\**'
```

Comment:
Isn't the directory structure `*\*-*-*-*` instead of broad `**` ?

Reply:
never mind I might be confusing this with Defender

if you have a documentation link that could help

Reply:
I only have the path.